src/Compiler/Driver/CompilerConfig.fs

```diff
@@ -91,16 +91,23 @@ let ResolveFileUsingPaths (paths, m, fileName) =
         let searchMessage = String.concat "\n " paths
         raise (FileNameNotResolved(fileName, searchMessage, m))
 
-let GetWarningNumber (m, warningNumber: string) =
+let GetWarningNumber (m, warningNumber: string, prefixSupported) =
     try
         // Okay so ...
         //      #pragma strips FS of the #pragma "FS0004" and validates the warning number
         //      therefore if we have warning id that starts with a numeric digit we convert it to Some (int32)
         //      anything else is ignored None
```

Comment:
probably ancient, but... it is not `#pragma` anymore (F# pre 1.0 days?) it is `#nowarn` now. This line explains why, in the past, `FS` prefix was valid (auto-stripped), but not anymore. I suggest to drop this comment.

```suggestion
        // #nowarn may have a prefix 'FS' which we strip here, to keep only the number. 
        // Other prefixes return None
```

Reply:
done, thanks